test/unit/offset.js

```diff
@@ -165,9 +165,9 @@ testIframe( "absolute", "offset/absolute.html", function( assert, $ ) {
 
 	// get offset tests
 	tests = [
-		{ "id": "#absolute-1",     "top":  1, "left":  1 },
-		{ "id": "#absolute-1-1",   "top":  5, "left":  5 },
-		{ "id": "#absolute-1-1-1", "top":  9, "left":  9 },
+		{ "id": "#absolute-1",     "top": 1, "left": 1 },
+		{ "id": "#absolute-1-1",   "top": 5, "left": 5 },
+		{ "id": "#absolute-1-1-1", "top": 9, "left": 9 },
```

Comment:
If you're removing extra spaces here, let's be consistent and remove them before `"top"` as well. A similar remark applies to many places below as well.

Reply:
Technically, the eslint rule only applies to space *after* the key.

Reply:
I removed the extra spacing on index from the above comment. It didn't seem to serve a purpose anyway as nothing was lined up. These I don't mind as much. More importantly, we don't have a rule set to catch this, so I'm reticent to change it without eslint verification.